Sources/CoreCommands/Options.swift

```diff
@@ -464,10 +464,6 @@ package struct BuildOptions: ParsableArguments {
         #endif
     }
 
-    /// Whether to enable test discovery on platforms without Objective-C runtime.
```

Comment:
We may want to keep this flag in place and just not use it for anything except generating a diagnostic. At least for a little bit longer, to give affected packages like swift-numerics a chance to migrate.